File: java practical/p5.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
public class p5 {
    public static void main(String[] args) {
        System.out.println("Name : Sameer");
        System.out.println("Class : M.Sc(IT) - 3rdSem\n");
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        System.out.println("Please Enter Some Text : ");
        try {
            String userInput = reader.readLine();
            System.out.println("You Entered : " + userInput);
        } catch (IOException e) {
            System.out.println("An error occurred while trying to read input : " + e.getMessage());
        } finally{
            try{
                reader.close();
            }
            catch(IOException e){
                System.out.println("Failed to close the BufferedReader : " + e.getMessage());
            }
        }
    }
}

File: python practical/p5.py
print("Name : Sameer\nClass : M.Sc(IT) - 3rd sem\n")
num1 = int(input("Enter the number : "))
if num1 > 1:
    for i in range(2, num1):
        if num1 % i == 0:
            print(num1, "is not a prime number.")
            break
        else:
            print(num1, "is prime number.")
            break
else:
    print(num1, "is not a prime number.")
